src/evergreen/cli/main.py

```diff
@@ -371,6 +371,34 @@ def build_stats(ctx, build_id, tasks):
         click.echo(fmt_output(fmt, build.get_metrics().as_dict(include_children=tasks)))
 
 
+@cli.command()
+@click.pass_context
+@click.option("--project", required=True)
+@click.option("--commit", required=True)
+def manifest(ctx, project, commit):
+    """Get a manifest for the given project and commit."""
+    api = ctx.obj["api"]
+    fmt = ctx.obj["format"]
+
+    manifest = api.manifest(project, commit)
+    click.echo(fmt_output(fmt, manifest.json))
+
+
+@cli.command()
+@click.pass_context
+@click.option("--project", required=True)
+@click.option("--commit", required=True)
+@click.option("--module", required=True)
+def get_module_commit(ctx, project, commit, module):
```

Comment:
Do we really want to add this command? It seems like the same as the previous command, but then you are only extracting 1 piece of data from it. I don't think we would want to support separate commands for every possible data point that could be retrieved from the API. I wonder if it might be better to use something like [jq](https://stedolan.github.io/jq/) to extract the data you want from the previous command.

Reply:
I had the same thought. However, I only added the "manifest" command for completeness. My main use case is actually this "get-module-commit command", and it would be nice to have it as a self-sufficient command with no need for jq. I would rather keep both commands, or delete "manifest" and keep "get-module-commit", instead of deleting "get-module-commit" and use "manifest" via jq.

Reply:
I've added `help` parameters to these options.

Reply:
Why not add a flag to the `manifest` command to have it just display the manifest revision? Having a command to extract 1 piece of data is not a precedent that I want to set.

Reply:
I like the flag even less, so I've deleted the get-module-version command and kept only the "manifest" command, with some example help text showing how to use it with jq.